distribution/docs/src/main/jdocs/content/_developingComponents/developing-query-response-transformers-contents.adoc

```diff
@@ -0,0 +1,55 @@
+:title: Developing Query Response Transformers
+:type: developingComponent
+:status: published
+:link: _developing_query_response_transformers
+:summary: Creating a custom Query Response Transformer.
```

Comment:
Be consistent with the period usage in the summaries.
I'm neither for or against them.